sway/tree/container.c

```diff
@@ -573,16 +573,32 @@ void container_update_title_textures(struct sway_container *container) {
 }
 
 void container_calculate_title_height(struct sway_container *container) {
+	sway_log(SWAY_INFO, "===> container_calculate_title_height");
```

Comment:
I think most of this logging isn't  very useful, so to make this PR mergeable, you should remove it.

Other than that, is there anything here that's actually WIP anymore? The Alpine build failure doesn't seem related to the code.